kolibri/plugins/user/assets/src/views/SignInPage/FacilityModal.vue

```diff
@@ -3,9 +3,7 @@
   <KModal
     :title="$tr('facilitySelectionModalHeader')"
     :submitText="$tr('submitFacilitySelectionButtonPrompt')"
-    :cancelText="$tr('close')"
```

Comment:
don't need the translated string below

Reply:
It’s being used somewhere else via cross component translator. This should be cleaned up on develop though